pkg/asset/machines/azure/azuremachines.go

```diff
@@ -92,7 +106,8 @@ func GenerateMachines(platform *azure.Platform, pool *types.MachinePool, userDat
 	additionalCapabilities := &capz.AdditionalCapabilities{
 		UltraSSDEnabled: &ultrassd,
 	}
-	if pool.Platform.Azure.DiskEncryptionSet != nil {
+	// XXX: What should we do with do with the passed in capabilities?
```

Comment:
Is this still an open question? I don't understand

Reply:
There is an unused parameter passed into the function named "Capabilities". What should we do with it? ;-)

Reply:
If they are unused, they can be removed. As part of this PR or separately.

Reply:
@jhixson74 can you remove this comment? I'll leave it up to you as to whether to remove the capabilities parameter or not.

Reply:
Removed comment and parameter. After looking at the code, everything in capabilities was being set prior to GenerateMachines() so it was not necessary to pass it. Fixed!